receiver/fluentforwardreceiver/receiver.go

```diff
@@ -59,7 +61,11 @@ func (r *fluentReceiver) Start(ctx context.Context, _ component.Host) error {
 
 	r.collector.Start(receiverCtx)
 
-	listenAddr := r.conf.ListenAddress
+	if r.conf.ListenAddress != nil {
+		listenAddr := r.conf.ListenAddress
+	} else {
+		listenAddr := defaultBindAddress
+	}
```

Comment:
You should not have this custom logic, instead change the factory `createDefaultConfig` to set the value by default.